tests/react/client/components/drawers/settings/index.test.tsx

```
@@ -0,0 +1,276 @@
+  submitShortcutApiToken: jest.fn()
+}))
+
+jest.mock('@/client/components/drawers/settings/google-auth-section', () => ({
+  GoogleAuthSection: jest.fn(({ onAuthStatusChange }) => {
+    // Simulate a button that can trigger auth status changes
+    return (
+      <div>
+        <h3>Step 1: Authenticate with Google</h3>
+        <button
+          data-testid="mock-google-auth-button"
+          onClick={() => onAuthStatusChange && onAuthStatusChange('success')}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)